fedimint-wasm-tests/src/lib.rs

```diff
@@ -208,4 +212,43 @@ mod tests {
 
         Ok(())
     }
+
+    async fn send_and_recv_ecash_once(
+        client: fedimint_client::ClientArc,
+    ) -> Result<(), anyhow::Error> {
+        let mint = client.get_first_module::<MintClientModule>();
+        let (_, notes) = mint
+            .spend_notes(Amount::from_sats(11), Duration::from_secs(10000), false, ())
```

Comment:
I've noticed 10 seconds for `try_cancel_after` in other mint tests. Would it make sense to reduce here?

Reply:
This is for cancellation that shouldn't be needed here and isn't waited for. I'd be a bit worried that it's too little and under heavy load possibly it could fail due to it?

Reply:
no, we never want client to cancel the ecash here, because are reissuing it in very next step.